### PR TITLE
fixcase1

### PR DESCRIPTION
**Pull request by paulflang**
Unit test 1 requires deletion of the events in the SBML (or elsewhere). We deleted in the SBML.

### CODE REVIEW

**Review by YingboMa — APPROVED** [review]

File: docs/src/Scenario2/Evaluation_Scenario_2.md [hunk 1]
```diff
@@ -49,14 +49,21 @@ ssys = structural_simplify(sys2)
...
+probne = ODEProblem(ssysne, [], (0.0, 100.0))
 ITALY_POPULATION = 60e6
 idart = [Infected, Diagnosed, Ailing, Recognized, Threatened]
-xmax, xmaxval = get_max_t(prob, sum(idart) * ITALY_POPULATION)
+xmax, xmaxval = get_max_t(probne, sum(idart))
 
-@test isapprox(xmax, 47; atol = 5)
-@test_broken isapprox(xmaxval, 0.6)
+@test isapprox(xmax, 47; atol = 1)
+@test isapprox(xmaxval, 0.6, atol=1)
```
Comment (Keno) [inline docs/src/Scenario2/Evaluation_Scenario_2.md:66]: `atol=1` on a percentage value seems problematic. Doesn't that allow any value?

File: Scenario2/bin.jl [hunk 2]
```diff
@@ -22,6 +22,7 @@ using OrdinaryDiffEq, ModelingToolkit, EasyModelAnalysis, SBML, SBMLToolkit, UnP
 import Plots
 
 fn = "Giordano2020.xml"
+fn = joinpath("..", "docs", "src", "Scenario2", "Giordano2020.xml")
```
[the remark this answers is not in the document]
Comment (ChrisRackauckas) [reply Scenario2/bin.jl:25]: That would break the build system anyways. It should be sourced from the docs/src like all input data/images

Comment (ChrisRackauckas) [inline Scenario2/bin.jl:25]: But this isn't using the noevs.xml one?

File: docs/src/Scenario2/Evaluation_Scenario_2.md [hunk 3]
```diff
@@ -49,14 +49,21 @@ ssys = structural_simplify(sys2)
 
 > Simulate for 100 days, and determine the day and level of peak total infections (sum over all the infected states I, D, A, R, T). Expected output: The peak should occur around day 47, when ~60% of the population is infected.
 
+This unit test requires deletion of the events in the SBML (or elsewhere). We deleted in the SBML.
```
Comment (Keno) [inline docs/src/Scenario2/Evaluation_Scenario_2.md:52]: Can we elaborate on why? What's causing issues in the presence of these events? What do the events mean, etc...

File: docs/src/Scenario2/Evaluation_Scenario_2.md (hunk 1 above)
Comment (YingboMa) [reply docs/src/Scenario2/Evaluation_Scenario_2.md:66]: See https://github.com/ChrisRackauckas/ASKEM_Evaluation_Staging/pull/35

**Pull request closed without merging**